tests/functional/ingestion/S3Mock.js

```diff
@@ -0,0 +1,96 @@
+const async = require('async');
+const AWS = require('aws-sdk');
+// const http = require('http');
+
+// const { MetadataMock } = require('arsenal').testing.MetadataMock;
+const testConfig = require('../../config.json');
+const BackbeatClient = require('../../../lib/clients/BackbeatClient');
+
+const sourceConfig = testConfig.extensions.ingestion.sources[0];
```

Comment:
instead of declaring sourceConfig here, can you pass it as a param for setupS3Mock